.gitignore

```diff
@@ -40,3 +41,6 @@ sbt_cache/*
 **/.ammonite/
 **/metals.sbt
 **/pom.xml
+universal-application-tool-0.0.1/package-lock.json
```

Comment:
issue (blocking): why ignore the package lock file? without it package versions will change without us noticing